app/views/search/results.njk

```diff
@@ -51,9 +51,13 @@
         page: currentPage,
         searchString: searchString
       }) }}
-      {{ govukPagination({
+
+     {{ govukPagination({
         previous: previous,
-        next: next
+        next: next,
+        attributes: {
+          "aria-label": 'Pagination Navigation'
+        }
```

Comment:
This will work but since we are passing previous and next from the ts file, it would be better to pass the attributes from the same function. Once you have added the attributes object, use it here like:
`next: next,
       attributes: attributes`

Reply:
updated the code as per comments